src/components/OptionRow.js

```diff
@@ -97,232 +97,217 @@ const defaultProps = {
     shouldDisableRowInnerPadding: false,
 };
 
-class OptionRow extends Component {
-    constructor(props) {
-        super(props);
-        this.state = {
-            isDisabled: this.props.isDisabled,
-        };
-    }
+function OptionRow(props) {
+    const [isDisabled, setIsDisabled] = useState(props.isDisabled);
 
-    // It is very important to use shouldComponentUpdate here so SectionList items will not unnecessarily re-render
-    shouldComponentUpdate(nextProps, nextState) {
-        return (
-            this.state.isDisabled !== nextState.isDisabled ||
-            this.props.isDisabled !== nextProps.isDisabled ||
-            this.props.isMultilineSupported !== nextProps.isMultilineSupported ||
-            this.props.isSelected !== nextProps.isSelected ||
-            this.props.shouldHaveOptionSeparator !== nextProps.shouldHaveOptionSeparator ||
-            this.props.selectedStateButtonText !== nextProps.selectedStateButtonText ||
-            this.props.showSelectedState !== nextProps.showSelectedState ||
-            this.props.highlightSelected !== nextProps.highlightSelected ||
-            this.props.showTitleTooltip !== nextProps.showTitleTooltip ||
-            !_.isEqual(this.props.option.icons, nextProps.option.icons) ||
-            this.props.optionIsFocused !== nextProps.optionIsFocused ||
-            this.props.option.text !== nextProps.option.text ||
-            this.props.option.alternateText !== nextProps.option.alternateText ||
-            this.props.option.descriptiveText !== nextProps.option.descriptiveText ||
-            this.props.option.brickRoadIndicator !== nextProps.option.brickRoadIndicator ||
-            this.props.option.shouldShowSubscript !== nextProps.option.shouldShowSubscript ||
-            this.props.option.ownerAccountID !== nextProps.option.ownerAccountID ||
-            this.props.option.subtitle !== nextProps.option.subtitle ||
-            this.props.option.pendingAction !== nextProps.option.pendingAction ||
-            this.props.option.customIcon !== nextProps.option.customIcon
-        );
-    }
+    useEffect(() => {
+        setIsDisabled(props.isDisabled);
+    }, [props.isDisabled]);
 
-    componentDidUpdate(prevProps) {
-        if (this.props.isDisabled === prevProps.isDisabled) {
-            return;
-        }
+    let pressableRef = null;
```

Comment:
I think the ref gets set after the component mounts. And we need to use `useRef()`. 

Otherwise, I think when `OptionRow` re-renders this would be `null` and the `ref` won't get set.

Reply:
Yeah, I'm sorry I missed it.
Now updated